workers/generic-worker/helper_test.go

```diff
@@ -58,14 +59,14 @@ func testWorkerType() string {
 	return "test-" + strings.ToLower(strings.Replace(slugid.Nice(), "_", "", -1)) + "-a"
 }
 
-func scheduleTask(t *testing.T, td *tcqueue.TaskDefinitionRequest, payload GenericWorkerPayload) (taskID string) {
+func scheduleTask[P GenericWorkerPayload | dockerworker.DockerWorkerPayload](t *testing.T, td *tcqueue.TaskDefinitionRequest, payload P) (taskID string) {
```

Comment:
Could we use `DockerWorkerPayload` here instead of `dockerworker.DockerWorkerPayload`?

Reply:
`DockerWorkerPayload` is undefined for the other schemas/engines though. It would only be defined for multiuser posix.